saleor/settings.py

```diff
@@ -627,6 +627,10 @@ def get_url_from_env(name, *, schemes=None) -> Optional[str]:
         "task": "saleor.payment.tasks.transaction_release_funds_for_checkout_task",
         "schedule": timedelta(minutes=10),
     },
+    "recalculate-discounted-price-for-products": {
+        "task": "saleor.product.tasks.recalculate_discounted_price_for_products_task",
+        "schedule": timedelta(minutes=20),
```

Comment:
Is 20 minutes correct here? 
I mean: It means that we will not update the prices for max 20 minutes (or even more if the amount of products to process is big).
By making a promotion, you could expect that undiscounted prices will show up immediately. Did we document it somewhere?

Reply:
I agree, IMO is too rare

Reply:
But if this is too rare,  we can end up with a huge amount of the same celery tasks which will occupy most of the available workers. 
We should have a validation that only one task is under processing. Or provide separate workers to only update the prices.  (we need to define which way is better and safer).